assets/CommonAssemblyInfo.cs

```diff
@@ -1,5 +1,5 @@
 using System.Reflection;
 
 [assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.1.1.1")]
+[assembly: AssemblyFileVersion("1.4.204.0")]
```

Comment:
When you build using `Build.ps1` locally, use the `-notouch` parameter to avoid inadvertently changing this file. Needs to be reverted from the commit.